uml/state_diagrams/settings.plantuml

```diff
@@ -0,0 +1,64 @@
+@startuml
+
+skinparam shadowing false
+
+title Settings State Diagram
+
+state "Settings Selection" as Ss{
+    state "Brightness Selection" as Bs {
+
+        state "Low Brightness" as Lb : entry: Set Brightness at 25%
+        state "Mid Brightness" as Mb : entry: Set Brightness at 50%
+        state "High Brightness" as Hb : entry: Set Brightness at 75%
+        state "Very High Brightness" as Vhb : entry: Set Brightness at 100%
+
+        [*]->Mb
+        Lb -up-> Mb : Press\nButton A
+        Mb -right-> Hb : Press\nButton A
+        Hb -down-> Vhb : Press\nButton A
+        Vhb -left-> Lb : Press\nButton A
+    }
+
+    state "Bluetooth Selection" as Bts {
+
+        state "Bluetooth On" as Ba : entry: Activate Bluetooth
+        state "Bluetooth Off" as Bd : entry: Deactivate Bluetooth
+
+        [*]->Ba
+        Ba -down-> Bd : Press\nButton A
+        Bd -up-> Ba : Press\nButton A
+    }
+
+    state "Health Check Notification" as Hcn {
+
+        state "Notification On" as Na : entry: Activate Notification
+        state "Notification Off" as Nd : entry: Deactivate Notification
+
+        [*]->Na
+        Na -down-> Nd : Press\nButton A
+        Nd -up-> Na : Press\nButton A
+    }
+
+    Bs : entry: Activate Brightness Selection Menu\nexit: Deactivate Brightness Selection Menu
+    Bts : entry: Activate Bluetooth Selection Menu\nexit: Deactivate Bluetooth Selection Menu
+    Hcn : entry: Activate Health Check Notification Selection Menu\nexit: Deactivate Health Check Notification Selection Menu
+
+    
+    Bs -right-> Bts : Press\nButton B
+    Bts -down-> Hcn : Press\nButton B
+    Hcn -up-> Bs : Press\nButton B
+}
+
+state "Settings Storage" as Sstor{
+
+
+}
+
+Sstor : exit: Store Selected Settings
+
+[*]-down->Bs : Press\nButton B
+Ss-down->Sstor : Hold\nButton B\nfor 3 secs
+Ss : entry: Activate Settings Selection Menu\nexit: Deactivate Settings Selection Menu
+Sstor->[*]
+
+@enduml
```

Comment:
As done in display view menu, button A is changing the selection, button B is activating

Reply:
Fixed